backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -56,39 +56,35 @@
   private static final String PROPERTY_FIRSTPARTICIPANT = "firstParticipant";
   private static final String PROPERTY_SECONDPARTICIPANT = "secondParticipant";
 
+  // HTTP Request JSON key constants
+  private static final String KEY_TIMEAVAILABLEUNTIL = "timeAvailableUntil";
+  private static final String KEY_DURATION = "duration";
+  private static final String KEY_ROLE = "role";
+  private static final String KEY_PRODUCTAREA = "productArea";
+  private static final String KEY_SAVEPREFERENCE = "savePreference";
+  private static final String KEY_MATCHPREFERENCE = "matchPreference";
```

Comment:
I know Key is technically the correct term for JSON keys, but I called them JSON_DURATION to differentiate between Datastore Keys.

Reply:
Thanks for the suggestion, done!